server/sendScreenshot.js

```diff
@@ -22,6 +22,24 @@ const getBufferFromRequest = (url) => {
   })
 }
 
+const getCroppedScreenshot = async (page, firstImageUrl) => {
```

Comment:
mostly copy-pasted from https://github.com/nickells/jeremy-the-bot/commit/417be78800c7498bb0107e49e6722300810cc92b#diff-01a6cefc3183806d1dee00bf0c4ec1bed99a97bb1383f8519f433612fcdf4d72